tests/queries/0_stateless/02163_storage_add_support_ttl.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS 02163_test_tb;
+
+CREATE TABLE 02163_test_tb (id UInt64, name String, dt Date) Engine=MergeTree ORDER BY id;
```

Comment:
>OK, but I'm not quite sure if the current tests are enough.

Not enough, since this is only covers MergeTree table. You need to also add a test for others, i.e. Distributed/Merge/Buffer/Memory/Log/TinyLog

stateless tests support jinja2 templates (.sql.j2/.reference.j2), so you can either use it (to avoid copy-paste), or write test in bash (.sh)

Reply:
Ok, but when I receive `Engine Distributed doesn't support TTL`, how should I make the test pass, I need some hints.

Reply:
Sure:

```sql
DROP TABLE IF EXISTS null_02163;
CREATE TABLE null_02163 AS system.one Engine=Null();
ALTER TABLE null_02163 MODIFY COLUMN dummy Int TTL now() + INTERVAL 1 MONTH; -- { serverError BAD_ARGUMENTS }
DETACH TABLE null_02163;
ATTACH TABLE null_02163;
```

(for tests clickhouse-client supports some hints, like `serverError` - https://github.com/ClickHouse/ClickHouse/blob/dd2779fadc1874846b52839ef18244aedb8636fc/src/Client/TestHint.h#L10-L42)